src/components/StyleGuide/TextCase.mdx

```diff
@@ -0,0 +1,150 @@
+import { Meta, Source, Unstyled } from "@storybook/addon-docs/blocks";
+
+import Banner from "../Banner/Banner";
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
+import Icon from "../Icons/Icon";
+import Link from "../Link/Link";
+import Table from "../Table/Table";
+
+<Meta title="Style Guide/Text Case" />
+
+<ComponentDocsHeader
+  category="Style Guide"
+  componentName="Text case"
+  summary="Guidelines for applying capitalization to text elements"
+/>
+
+## Table of contents
+
+- [General guidance](#general-guidance)
+- [Title case](#title-case)
+- [All uppercase](#all-uppercase)
+
+## General guidance
+
+All text elements&mdash;including, but not limited to, page titles, headings,
+labels, menu items, navigation components, and buttons&mdash;should use sentence
+case where only the first letter of the first word in a phrase is capitalized.
+
+<Unstyled>
+  <Table
+    columnHeaders={["Element", "Correct format", "Incorrect format"]}
+    fontSize="desktop.body.body2"
+    borderTop="1px solid"
+    borderTopColor="ui.gray.semi-medium"
+    tableData={[
+      [
+        "Heading",
+        <>
+          Prints and drawings{" "}
+          <Icon
+            align="right"
+            color="ui.success.primary"
+            name="check"
+            size="medium"
+          />
+        </>,
+        <>
+          Prints and Drawings
+          <Icon
+            align="right"
+            color="ui.error.primary"
+            name="close"
+            size="medium"
+          />
+        </>,
+      ],
+      [
+        "Menu item",
+        <>
+          Get help
+          <Icon
+            align="right"
+            color="ui.success.primary"
+            name="check"
+            size="medium"
+          />
+        </>,
+        <>
+          Get Help
+          <Icon
+            align="right"
+            color="ui.error.primary"
+            name="close"
+            size="medium"
+          />
+        </>,
+      ],
+      [
+        "Menu item",
+        <>
+          See all results
+          <Icon
+            align="right"
+            color="ui.success.primary"
+            name="check"
+            size="medium"
+          />
+        </>,
+        <>
+          See All Results
+          <Icon
+            align="right"
+            color="ui.error.primary"
+            name="close"
+            size="medium"
+          />
+        </>,
+      ],
+    ]}
+    useRowHeaders
+    showRowDividers
+  />
+</Unstyled>
+
+When using sentence case, don't put a period at the end of a title or heading.
+
+This guidance for sentence case applies to all text unless otherwise specified.
```

Comment:
Should this be moved to the start of the section? If not, it feels redundant to the first message at the top.